frontend/public/reducers/ui.ts

```diff
@@ -43,6 +43,30 @@ const newQueryBrowserQuery = (): ImmutableMap<string, any> =>
     isExpanded: true,
   });
 
+const getFiringAlerts = (alerts) =>
+  _.filter([...[].concat(alerts || [])], (a) => {
```

Comment:
Why is this `[...[].concat(alerts || [])]`? Looks like just `alerts` would be OK?

Reply:
How about using an `isAlertFiring` function to pass to the filter function instead of using `getFiringAlerts`?

```
const isAlertFiring = (alert) =>
  alert?.state === AlertStates.Firing || alert?.state === AlertStates.Silenced;
```

`const firingAlerts = _.filter(alerts?.data, isAlertFiring);`

`silenceFiringAlerts(_.filter(notificationAlerts?.data, isAlertFiring), silences);`